validator/spec/Fuzz.spec.yml

```diff
@@ -3,3 +3,32 @@
   query: '{r{__typename(s:0)}}'
   errors:
     - message: Cannot query field "r" on type "QueryRoot".
+
+- name: 02 - infinit loop occured in OverlappingFieldsCanBeMerged rule
+  schema: |
+    type Query {
+      fieldA: A
+    }
+    type A {
+      s: String
+    }
+  query: |
+    { ...F }
+    fragment F on Query {
+      fieldA { s }
+      ...{
+        ...{
+          ...F
+        }
+        fieldA {
+          ...notExists
+        }
+      }
+    }
+  errors:
+    # from KnownFragmentNames rule
```

Comment:
I found another issue(and not try to fix it yet
KnownFragmentNames report too much error message.